docs/web_reference.rst

```diff
@@ -1413,6 +1413,8 @@ duplicated like one using :meth:`Application.copy`.
       The table is a :class:`list` of :class:`RouteDef` items or
       :class:`RouteTableDef`.
 
+      Returns a :class:`list` of registered :class:`AbstractRoute` instances.
```

Comment:
1. Please use `:returns:` markup as in the method above.
2. Please add `.. versionchanged:: 3.7` markup, search in docs for the directive usage. The master is not ready for 4.0 release yet, I think we can publish 3.7 earlier.